src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -359,6 +359,214 @@ public void ModifyMethod_WithTuples()
                 Handle(2, TableIndex.AssemblyRef));
         }
 
+        [Fact]
+        public void ModifyMethod_WithAttributes()
+        {
+            var source0 =
+@"class C
+{
+    static void Main() { }
+    [System.ComponentModel.Description(""The F method"")]
+    static string F() { return null; }
+}";
+            var source1 =
+@"class C
+{
+    static void Main() { }
+    [System.ComponentModel.Description(""The F method"")]
+    static string F() { return string.Empty; }
+}";
+            var source2 =
+@"class C
+{
+    static void Main() { }
+    [System.ComponentModel.Description(""The F method""), System.ComponentModel.Category(""Methods"")]
```

Comment:
>[System.ComponentModel.Description(""The F method""), System.ComponentModel.Category(""Methods"")] [](http://example.com/codeflow?start=4&length=98)

Consider testing the case where the new attribute is inserted ahead of the existing attribute.

Reply:
Yep, good idea. This would also be a good reason to improve the test verification to check not just that there is a CustomAttribute edit for row 4, but which attribute type it actually is.